recipes/xenonpy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "XenonPy" %}
+{% set version = "0.1.0b1" %}
+{% set md5 = "bc96fc44c69c5164707c635e3e6de63b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/yoshida-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script:
+    - pip install pymatgen matminer
```

Comment:
Yeah, you can't do this unfortunately. Will have to package `pymatgen` and `matminer` in conda first.